Dockerfile_build

```diff
@@ -17,7 +17,7 @@ COPY Pipfile* /root/
 WORKDIR /root
 
 RUN pipenv install --system \
-    && sed -i 's|/bin/sh|/bin/bash|g' /usr/local/lib/python3.8/site-packages/testinfra/backend/docker.py
+    && sed -i 's|/bin/sh|/bin/bash|g' /usr/local/lib/python3.9/site-packages/testinfra/backend/docker.py
```

Comment:
This needs to be set back to 3.8, the `DOCKERFILE_build` in the dev branch is now pinned to 3.8

Reply:
Changed back to 3.8